### PR TITLE
Removed shared and packed layouts tests.

### PR DESCRIPTION
**Pull request by jasomobica**
Signed-off-by: Janusz Sobczak <janusz.sobczak@mobica.com>

### CODE REVIEW

**Comment by kenrussell** [comment]
Thanks, these changes LGTM.

I think that it may be difficult long term to track the differences between the WebGL versions of these tests and the vanilla ES3 tests.

@Oletus @jdashg @toji @zhenyao do any of you have comments on these test changes?

File: sdk/tests/deqp/functional/gles3/es3fUniformBlockTests.js [hunk 1]
```diff
@@ -393,7 +393,7 @@ goog.scope(function() {
         ];
 
         /** @type {Array<Object.<string, glsUniformBlockCase.UniformFlags>>} */
-        var layoutFlags = [{ name: 'shared', flags: glsUniformBlockCase.UniformFlags.LAYOUT_SHARED },
+        var layoutFlags = [ /* { name: 'shared', flags: glsUniformBlockCase.UniformFlags.LAYOUT_SHARED }, */
```
Comment (Oletus) [inline sdk/tests/deqp/functional/gles3/es3fUniformBlockTests.js]: Why is this commented out rather than just removed?

Comment (jasomobica) [inline sdk/tests/deqp/functional/gles3/es3fUniformBlockTests.js]: I left it commented out because this code could be potentially useful to somebody.

**Comment by kenrussell** [comment]
Merging now. Thanks for cleaning this up.

**Pull request merged**